flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/FsStreamingSinkITCaseBase.scala

```diff
@@ -50,84 +52,151 @@ abstract class FsStreamingSinkITCaseBase extends StreamingTestBase {
-                 |)
-                 |${if (partition) "partitioned by (d, e)" else ""}
-                 |with (
-                 |  'connector' = 'filesystem',
-                 |  'path' = '$resultPath',
-                 |  '${PARTITION_TIME_EXTRACTOR_TIMESTAMP_PATTERN.key()}' =
-                 |      '${dollar}d ${dollar}e:00:00',
-                 |  '${SINK_PARTITION_COMMIT_DELAY.key()}' = '1h',
-                 |  '${SINK_PARTITION_COMMIT_POLICY_KIND.key()}' = '$policy',
-                 |  '${SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME.key()}' = '_MY_SUCCESS',
-                 |  ${additionalProperties().mkString(",\n")}
-                 |)
-       """.stripMargin
-    tEnv.executeSql(ddl)
-
-    tEnv.sqlQuery("select * from my_table").executeInsert("sink_table").await()
-
-    check("select * from sink_table", data)
-  }
-
   @Test
   def testMetastorePolicy(): Unit = {
     thrown.expectMessage(
       "Can not configure a 'metastore' partition commit policy for a file system table." +
-          " You can only configure 'metastore' partition commit policy for a hive table.")
-    test(partition = true, "metastore")
+        " You can only configure 'metastore' partition commit policy for a hive table.")
+    testPartitionCustomFormatDate(partition = true, "metastore")
   }
+  
+  @Test
+  def testPartitionWithBasicDate(): Unit = {
+
+    // create source test data stream
+    val fun = (t: Row) => {
+      val localDateTime = LocalDateTime.of(
+        LocalDate.parse(t.getFieldAs[String](3), DateTimeFormatter.BASIC_ISO_DATE),
+        LocalTime.MIDNIGHT)
+      TimestampData.fromLocalDateTime(localDateTime).getMillisecond
+    }
+
+    val stream: DataStream[Row] = new DataStream(env.getJavaEnv.addSource(
+      new FiniteTestSource(data2, fun),
+      new RowTypeInfo(Types.INT, Types.STRING, Types.STRING, Types.STRING, Types.STRING)))
+
+    // write out the data
+    test(stream, "default", "yyyyMMdd", "$d", "d", "partition-time", "1d", data2)
 
+    // verify that the written data is correct
+    val basePath = new File(new URI(resultPath).getPath)
+    Assert.assertEquals(2, basePath.list().length)
+    Assert.assertTrue(new File(new File(basePath, "d=20200503"), "_MY_SUCCESS").exists())
+    Assert.assertTrue(new File(new File(basePath, "d=20200504"), "_MY_SUCCESS").exists())
+  }
+
+  def testPartitionCustomFormatDate(partition: Boolean, policy: String = "success-file"): Unit = {
+
+    val fun = (t: Row) => {
+      val localDateTime = LocalDateTime.parse(s"${t.getField(3)} ${t.getField(4)}:00:00",
+        DateTimeFormatter.ofPattern("MM-dd-yyyy HH:mm:ss"))
+      TimestampData.fromLocalDateTime(localDateTime).getMillisecond
+    }
+
+    val stream = new DataStream(env.getJavaEnv.addSource(
+      new FiniteTestSource(data, fun),
+      new RowTypeInfo(Types.INT, Types.STRING, Types.STRING, Types.STRING, Types.STRING)))
+
+    test(stream, "default", "", "$d $e:00:00", if (partition) "d,e" else "",
```

Comment:
IIUC, we should pass a `timeExtractorFormatterPattern` to check what you change here?

Reply:
done